src/Php80/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "symfony/polyfill-php80",
+    "type": "library",
+    "description": "Symfony polyfill backporting some PHP 8.0+ features to lower PHP versions",
+    "keywords": ["polyfill", "shim", "compatibility", "portable"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Ion Bazan",
+            "email": "ion.bazan@gmail.com"
+        },
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.3"
```

Comment:
If this needs php 7 to function correctly, then should this be bumped as well?

Reply:
That would prevent you from installing `symfony/polyfill` meta package on PHP 5.

Reply:
PHP 7 here yes. The root json file is a separate one so no issue with it.

Reply:
@nicolas-grekas I bumped the version but I wonder if we should do that now. I'm planning to submit another PR with `ValueError` class which could potentially be used on PHP 5 using `symfony/polyfill-php70` which includes `*Error` classes.